glue/flagging_script_glue/flagging.py

```diff
@@ -197,6 +197,46 @@ def pricing_info(
     if not condos:
         prices.insert(1, f"sv_price_per_sqft_deviation_{group_string}")
 
+    # Persist standard deviation per group
+    group_std = (
+        df.groupby(list(groups), group_keys=False)["meta_sale_price"]
+        .std(ddof=0)
```

Comment:
**suggestion:** I would add a quick in-line comment here so nobody else gets tripped up by the numpy/pandas difference in the future.